lib/ai/model/prompt.go

```diff
@@ -100,3 +103,16 @@ USER'S INPUT
 
 Okay, so what is the response to my last comment? If using information obtained from the tools you must mention it explicitly without mentioning the tool names - I have forgotten all TOOL RESPONSES! Remember to respond with a markdown code snippet of a json blob with a single action, and NOTHING else.`, toolResponse)
 }
+
+func MessageClassificationPrompt(classes map[string]string) string {
+	var prompt strings.Builder
```

Comment:
You really don't need the string builder for that. Look at the other prompts. It's easier to read them when you have the whole prompt as a single string.